selfdrive/car/toyota/carstate.py

```diff
@@ -226,7 +229,7 @@ def get_can_parser(CP):
       signals.append(("INTERCEPTOR_GAS2", "GAS_SENSOR"))
       checks.append(("GAS_SENSOR", 50))
 
-    if CP.enableBsm:
+    if CP.enableBsm or (CP.enableBsm and CP.carFingerprint in (CAR.CHR, CAR.CHRH)):
```

Comment:
I don't think this does anything

Reply:
Okay yea just realised it as well, that's an error on my part, will fix